specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -231,6 +232,7 @@ suppressions:
     reason: There is a similar usage in existing jobs.
     where:
       - $.definitions["FineTuningJob"].properties["outputs"]
+      - $.definitions["Registry"].properties["syndicatedRegistries"]
```

Comment:
>syndicatedRegistries [](http://example.com/codeflow?start=46&length=20)

I did not find previous instances of this exact suppression for syndicatedRegistries. So please come to the API review office hours to explain why this new scenario requires this anti-pattern

Reply:
aka.ms\armofficehoursinfo

Reply:
pl remove this as u are no longer using additional properties